v3/integrations/nropenai/nropenai.go

```diff
@@ -8,18 +8,23 @@ import (
 	"errors"
 	"reflect"
 	"strings"
+	"time"
 
 	"github.com/google/uuid"
+	"github.com/newrelic/go-agent/v3/internal"
 	"github.com/newrelic/go-agent/v3/newrelic"
 	"github.com/sashabaranov/go-openai"
 )
 
+func init() { internal.TrackUsage("Go", "ML", "OpenAI", "1.20.2") }
```

Comment:
If I understand the spec correctly, this should be reporting the version of the library the customer actually linked with their code, not a hardcoded value.